core/app/[locale]/not-found.tsx

```diff
@@ -32,6 +32,7 @@ const NotFoundQuery = graphql(
 
 export default async function NotFound() {
   const t = await getTranslations('NotFound');
+  const ct = await getTranslations('Components.Header.MiniCart');
```

Comment:
Would love to know if there are performance implications doing this double `await` for translations. 

Also, I think our pattern for things like this is just throw another translation string in the `NotFound` namespace instead of creating a new translation getter.